support.html

```diff
@@ -41,7 +41,7 @@
 	<link rel="stylesheet" href="css/responsive.css">
 	<link rel="stylesheet" href="js/sweetalert2//dist/sweetalert2.min.css">
 	<script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
-	<script src='https://www.google.com/recaptcha/api.js'></script>
+	<script src="https://www.google.com/recaptcha/api.js?render=6LeYMbcZAAAAAGW_wktwCx5gZe1qDnkLFNNij2GI"></script>
```

Comment:
Why are you not using the provided site_key in your Pull Requests?
It includes localhost as an option, so no need to use anything different even during your testing.